packages/quantic/force-app/main/default/lwc/quanticRecommendationList/e2e/quanticRecommendationList.e2e.ts

```diff
@@ -0,0 +1,36 @@
+import {testSearch} from './fixture';
+
+const fixtures = {
+  search: testSearch,
+};
+
+let test = fixtures.search;
+
+test.describe('quantic recommendation list', () => {
+  test.describe('when clicking a recommendation', () => {
+    test('should log recommendation open analytics', async ({
+      recommendationList,
+      page,
+    }) => {
+      // Hack(?) to be able to read the payload of the analytics click event.
+      await page.route('*analytics*', (route) => {
+        route.continue();
+      });
```

Comment:
I liked how @erocheleau used this hack, please flow him for a better organization:  https://github.com/coveo/ui-kit/blob/d7198f382d405a9d19cf9302ef1a012900bb62b1/packages/quantic/force-app/main/default/lwc/quanticResultList/e2e/pageObject.ts#L41